Source/WebCore/bindings/js/JSEventListener.cpp

```diff
@@ -47,10 +47,11 @@
 namespace WebCore {
 using namespace JSC;
 
-JSEventListener::JSEventListener(JSObject* function, JSObject* wrapper, bool isAttribute, CreatedFromMarkup createdFromMarkup, DOMWrapperWorld& isolatedWorld)
+JSEventListener::JSEventListener(JSObject* function, JSObject* wrapper, bool isAttribute, CreatedFromMarkup createdFromMarkup, DOMWrapperWorld& isolatedWorld, bool isWindowOnError = false)
```

Comment:
Perhaps it's different for JSC, but generally the preference is that boolean arguments use a `class enum`. That makes it more readable from the caller side what is going on.